api-examples/pretrain_paired_pytorch.py

```diff
@@ -80,7 +80,8 @@ def train():
     parser.add_argument("--num_layers", type=int, default=8, help="Number of layers")
     parser.add_argument("--windowed_ra", type=str2bool, default=False, help="whether prevent attention beyond rpr_k")
     parser.add_argument("--num_train_workers", type=int, default=4, help="Number train workers")
-    parser.add_argument("--nctx", type=int, default=256, help="Max input length")
+    parser.add_argument("--src_nctx", type=int, default=256, help="Max input length")
```

Comment:
I would suggest leaving nctx and offering an optional tgt_nctx.  If its not provided use nctx

Reply:
done